libraries/persistent_store/src/lib.rs

```diff
@@ -12,299 +12,297 @@
+//!     [page](format::ERASE_PAGE) to be erased and a [checksum](format::WORD_CHECKSUM).
+//! -   [Clear](format::ID_CLEAR): A word used during the clear operation. It contains the
+//!     [threshold](format::CLEAR_MIN_KEY) and a [checksum](format::WORD_CHECKSUM).
+//! -   [Marker](format::ID_MARKER): A word used during a transaction. It contains the [number of
+//!     updates](format::MARKER_COUNT) following the marker and a [checksum](format::WORD_CHECKSUM).
+//! -   [Remove](format::ID_REMOVE): A word used inside a transaction. It contains the
+//!     [key](format::REMOVE_KEY) of the entry to be removed and a
+//!     [checksum](format::WORD_CHECKSUM).
 //!
 //! Checksums are the number of bits equal to 0.
 //!
 //! # Proofs
 //!
 //! ## Compaction
 //!
-//! It should always be possible to fully compact the store, after what the
-//! remaining capacity should be available in the current window (restoring the
-//! compaction invariant). We consider all notations on the virtual storage after
-//! the full compaction. We will use the `|x|` notation although we update the state
-//! of the virtual storage. This is fine because compaction doesn't change the
-//! status of an existing word.
+//! It should always be possible to fully compact the store, after what the remaining capacity
+//! should be available in the current window (restoring the compaction invariant). We consider all
+//! notations on the virtual storage after the full compaction. We will use the |x| notation
+//! although we update the state of the virtual storage. This is fine because compaction doesn't
+//! change the status of an existing word.
 //!
-//! We want to show that the next `N - 1` compactions won't move the tail past the
-//! last page of their window, with `I` the initial window:
+//! We want to show that the next N - 1 compactions won't move the tail past the last page of their
+//! window, with I the initial window:
 //!
-//! ```text
-//! forall 1 <= i <= N - 1, t_{I + i} <= (I + i + N - 1) * Q
-//! ```
+//! |                  |            |   |                     |
+//! | ----------------:| ----------:|:-:|:------------------- |
+//! | ∀(1 ≤ i ≤ N - 1) | t\_{I + i} | ≤ | (I + i + N - 1) × Q |
 //!
-//! We assume `i` between `1` and `N - 1`.
+//! We assume i between 1 and N - 1.
 //!
-//! One step of compaction advances the tail by how many words were used in the
-//! first page of the window with the last entry possibly overlapping on the next
-//! page.
+//! One step of compaction advances the tail by how many words were used in the first page of the
+//! window with the last entry possibly overlapping on the next page.
 //!
-//! ```text
-//! forall j, t_{j + 1} = t_j + |h_{j + 1}| - |h_j| + 1
-//! ```
+//! |    |            |   |                                      |
+//! | --:| ----------:|:-:|:------------------------------------ |
+//! | ∀j | t\_{j + 1} | = | t\_j + \|h\_{j + 1}\| - \|h\_j\| + 1 |
```

Comment:
Same here.

Reply:
Same, this particular table is aligned for me.